Tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -95,6 +95,7 @@ public function testEmptyConfiguration()
             'config/empty.yml',
             'config/empty.xml',
             'config/empty.php',
+            'config/toolbar.yml',
```

Comment:
maybe have a dedicated BC test that makes sure old configuration results in the right new configuration. then we can also easily remove that test when we bump the major version.